build.sbt

```diff
@@ -2,7 +2,7 @@ import Tests._
 
 // This gives us a nicer handle  to the root project instead of using the
 // implicit one
-lazy val chipyardRoot = RootProject(file("."))
+lazy val chipyardRoot = Project("chipyardRoot", file("."))
```

Comment:
Is this just cosmetic or does it change functionality?

Reply:
It makes it possible to reference the root project using a `ProjectRef` in FireSim.